src/edu/csus/ecs/pc2/core/Constants.java

```diff
@@ -96,18 +96,23 @@ private Constants() {
      * The Default PC2 Validator Command Line
      */
     public static final String DEFAULT_PC2_VALIDATOR_COMMAND = "{:validator} {:infile} {:outfile} {:ansfile} {:resfile} ";
-    
+
     /**
      * The Default CLICS Validator Command Line
      */
     public static final String DEFAULT_CLICS_VALIDATOR_COMMAND = "{:validator} {:infile} {:ansfile} {:feedbackdir} ";
-    
-    
+
+
     /**
      * Command substitution variable for name of thing to execute (basename of source file)
      */
     public static final String CMDSUB_BASENAME_VARNAME = "{:basename}";
-    
+
+    /**
+     * Command substitution variable for all files submitted. Note the missing ':' - legacy.
+     */
+    public static final String CMSSUB_FILES_VARNAME = "{files}";
```

Comment:
"CMSSUB..." seems to me to be a weird name.  Was the intention that it be "CMDSUB...", as in "Command Substitution"? If so, I'd like to see that corrected, for the sake of keeping code clean and clear...

Also, I haven't yet reviewed the rest of the code but:  there are just one or two places where `{files}` (without the colon) is used; all other places use the proper form (`{:files}`).  It seems weird to me to provide a "code solution" like this to fix something that is (almost certainly) a typo.  Why not just correct the typo in the one or two places it occurs?
And on a related note (again, I haven't reviewed the rest of the code changes), are there OTHER places that are now using this substitution constant (when they should be using `{:files}`) ?

Having said this, I can see filing a "clean-up" Issue that suggests defining a string like "CMDSUB_FILES_VARNAME" -- and then replacing all hard-coded instances of `{:files}` with that string....  But I don't like having a string like this that uses the WRONG value.  It's too possible for someone in the future to think "oh, there's a CONST for this substitution so I should use that" -- thus getting the WRONG substitution.  Brrrr... brittle code.

Reply:
Heh.  Yeah, that is a typo **CMSSUB** vs **CMDSUB**.
w/r/t `{files} `vs. `{:files}`, I was just concerned that `{files}` has been there so long (since 2017 when Doug added it), that if any CDP counts on `{files}` will break and any 
users that may use the GUI and "know" to use `{files}`, will now find it no longer works.  Are we concerned about "backward" compatibility for stuff that was wrong in the first place and was done 7 years ago?

Reply:
BTW, I actually mention https://github.com/pc2ccs/pc2v9/issues/869 in the PR description, which is what you are referring to.  Quite frankly, I'm inclined to leave {files} there and ADD {:files} as well, but others may have a different opinion.  (I hate to break things by unbreaking things).